src/timeline.js

```diff
@@ -542,6 +547,8 @@ export default class Timeline extends Component {
             data-row-index={rowIndex}
             className="rct9k-row"
             onClick={e => this._handleItemRowEvent(e, this.no_op, this.props.onRowClick)}
+            onMouseDown={e => this.selecting = false}
```

Comment:
Does this work ?

Reply:
Yeah seems to.
Should it not?

Reply:
dont know what it should do so checking you know what it does